.github/workflows/test.yml

```diff
@@ -10,12 +10,13 @@ on:
 
 jobs:
 
-  style:
-    name: Style
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-    - run: pipx run flake8 glass
+# TODO: replace with a more general pre-commit linter
+#   style:
+#     name: Style
+#     runs-on: ubuntu-latest
+#     steps:
+#     - uses: actions/checkout@v4
+#     - run: pipx run flake8 glass
```

Comment:
Unrelated to this PR, but maybe we can add pre-commit.ci to the repository?

Reply:
The problem is the changes invalidate `flake8` so it. kind of had to go

Reply:
I've never been a fan of the CI as it only works in public repos, but seeing as its public I guess we could

Reply:
Also could use the pre-commit or the pre-commit-lite action here.

Reply:
Will do in a different PR